ext/xmlrpc/xmlrpc-epi-php.c

```diff
@@ -963,15 +963,11 @@ PHP_FUNCTION(xmlrpc_server_call_method)
 	php_output_options out;
 	int argc = ZEND_NUM_ARGS();
 
-	if (zend_parse_parameters(argc, "Osz|a", &handle, xmlrpc_server_ce, &rawxml, &rawxml_len, &caller_params, &output_opts) != SUCCESS) {
+	if (zend_parse_parameters(argc, "Osz|a!", &handle, xmlrpc_server_ce, &rawxml, &rawxml_len, &caller_params, &output_opts) != SUCCESS) {
```

Comment:
As it's no longer marked nullable, the `!` shouldn't be here anymore.

Reply:
Ah, yes, of course. I fixed this with my recent amended commit.